rmw_zenoh_cpp/src/rmw_zenoh.cpp

```diff
@@ -775,6 +776,48 @@ rmw_return_loaned_message_from_publisher(
   return RMW_RET_UNSUPPORTED;
 }
 
+static z_owned_bytes_map_t create_map_and_set_sequence_num(
```

Comment:
Should we include this in an anonymous namespace?

Reply:
We aren't consistent here.  Sometimes we use the anonymous namespace, sometimes we use `static`.  They both accomplish the same thing.  I personally find the `static` to be easier to read (as you don't have to look at the namespace to find out whether it is a public function or not), but I don't feel strongly about it.

For now, I'm going to leave this as-is.  I'll open a separate issue to make this consistent throughout the codebase.